systemtest/loadgen/limiter.go

```diff
@@ -18,24 +18,21 @@
 package loadgen
 
 import (
-	"math"
+	"time"
 
 	"golang.org/x/time/rate"
 )
 
-func GetNewLimiter(epm int) *rate.Limiter {
-	if epm <= 0 {
+// Return a rate limiter that is configured to rate limit burst per interval.
```

Comment:
```suggestion
// GetNewLimiter returns a rate limiter that is configured to rate limit burst per interval.
```

It's conventional to start doc comments with the name of the method or type or whatever else you're documenting.